BEIMA.Backend/MongoService/MongoFilterGenerator.cs

```diff
@@ -27,5 +27,16 @@ public static FilterDefinition<BsonDocument> GetEqualsFilter(string key, dynamic
             }
             return Builders<BsonDocument>.Filter.Eq(key, value);
         }
+
+        /// <summary>
+        /// Combines two passed in filters. Will return one filter with the AND operation applied between the two separate filters.
+        /// </summary>
+        /// <param name="f1">The first filter.</param>
+        /// <param name="f2">The second filter.</param>
+        /// <returns>One filter that has the AND operation applied to both filters.</returns>
+        public static FilterDefinition<BsonDocument> CombineFilters(FilterDefinition<BsonDocument> f1, FilterDefinition<BsonDocument> f2)
```

Comment:
Instead of just limiting it to two at a time, why not add a params argument for the filter definitions here. That way we can use any number:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params